dev/intellij/install-runconfig.sh

```diff
@@ -16,6 +16,5 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 SRC_DIR="$SCRIPT_DIR/runConfigurations"
 DEST_DIR="$SCRIPT_DIR/../../.idea/runConfigurations/"
-echo `realpath $DEST_DIR`
```

Comment:
what's the impact of this?

Reply:
The main reason for removing this is that users may not have this realpath tool, and it will cause error. This line just show a message, so I think this has no effect.